.github/workflows/generate_go.yaml

```
@@ -1,9 +1,12 @@
-name: Test generating sources
+name: Generate Go stubs
 
-on: [push]
+on:
+  release:
+    types: [published]
 
 jobs:
   build:
+    if: startsWith(github.event.release.tag_name, 'go/v') == false
     runs-on: ubuntu-latest
 
```

```suggestion
    permissions:
      contents: write
```

The stefanzweifel/git-auto-commit-action needs write permissions. Ref:  https://github.com/stefanzweifel/git-auto-commit-action#usage